src/test/java/org/junitpioneer/jupiter/AbstractEntryBasedExtensionTests.java

```diff
@@ -16,8 +16,11 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledForJreRange;
+import org.junit.jupiter.api.condition.JRE;
 import org.junitpioneer.testkit.PioneerTestKit;
 
+@EnabledForJreRange(max = JRE.JAVA_16, disabledReason = "See: https://github.com/junit-pioneer/junit-pioneer/issues/509")
```

Comment:
On the last maintainer meeting Nicolai said that he'd prefer if we used the `--add-opens` option instead (since that's what is currently in our documentation).

Reply:
Hmmm, looks like the modular build doesn't know `org.junitpioneer`:

https://github.com/junit-pioneer/junit-pioneer/runs/7878861345?check_suite_focus=true#step:4:63

Will look into this.